tasks/eth/fp-recovery/001-blacklist-game/justfile

```diff
@@ -0,0 +1,17 @@
+set positional-arguments
+
+# default recipe to display help information
+default:
+  @just --list
+
+# Generate the `input.json` with a dispute game address to blacklist
+generate-input *args='':
```

Comment:
I'm not super familiar with `just`, can you drop a suggestion so I can see what this looks like?

Reply:
For _Generate the batch with `just generate-input <OptimismPortalProxyAddress> <DisputeGameToBlacklist>`._
```suggestion
generate-input OptimismPortalProxyAddress='' DisputeGameToBlacklist='' :
```
And then need to use the arguments `OptimismPortalProxyAddress` and `DisputeGameToBlacklist` inside the function by using `{{OptimismPortalProxyAddress}}`.
It would be nice to check if other tasks made can also be cleaned like this.